README.md

```diff
@@ -1,13 +1,15 @@
 # KanpurPython
-KanpurPython is one of the growing Python Community in India .We have completed 1 year 5 months in Kanpur and get awesome response from students and
-professional in Kanpur. We have 50+
-active members who works for spreading love of Python over Kanpur or region
-near by Kanpur . Recently we conduct two workshops in reputed colleges of
+KanpurPython is one of the growing Python Community in India. We recently completed our 1 year and 5 months. We received some great response from students as well as working professionals who live in and around Kanpu.
```

Comment:
We received some great response from students as well as working professionals who live in and around Kanpu. 
Kindly fix the typo